### PR TITLE
Normalize exercise file names. Fixes #15

### LINKED ISSUE #15: Normalize file names
@celavek I'd like to suggest that we follow @tmccombs's lead in how we name files:

```
$ tree exercises/gigasecond/
exercises/gigasecond/
├── gigasecond.d # this is a stub file, because it's an early exercise
├── gigasecond_example.d
└── gigasecond_test.d
```

let `$SLUG` represent the directory name, but with underscores instead of hyphens.

- `$SLUG_example.d`
- `$SLUG_test.d`

I'll prepare a PR with this now.

### PR DESCRIPTION
**Pull request by kytrinyx**
This makes an attempt at standardizing on the choices that @tmccombs made in #10.

@tmccombs @celavek Would I need to change some module names when I change the file names?

### CODE REVIEW

**Comment by celavek** [comment]
No. The module name is independent of the file name.

**Comment by celavek** [comment]
@tmccombs had some exercises that were different tham what I implemented. Are those part of this pull request?

**Comment by kytrinyx** [comment]
> Are those part of this pull request?

No, he already used this naming convention.

**Comment by kytrinyx** [comment]
I didn't understand the D idiom, and normalized in the wrong direction. I'll redo this.

**Comment by kytrinyx** [comment]
OK, I've normalized the filenames in all of the exercises.

**Comment by kytrinyx** [comment]
@celavek @tmccombs This is ready for review.

File: exercises/gigasecond/gigasecond.d [hunk 1]
```diff
@@ -1,8 +1,21 @@
-module gigasecond;
-
+import std.stdio;
 import std.datetime;
 
-DateTime gsAnniversary(DateTime start) {
-    // Your code here
-    return start;
+import gigasecond;
```
Comment (tmccombs) [inline exercises/gigasecond/gigasecond.d]: this should be removed

File: exercises/raindrops/raindrops.d [hunk 2]
```diff
@@ -1,6 +1,23 @@
-module raindrops;
+import std.stdio;
 
-string convert(int n) {
-    // your code here
-    return "";
+import raindrops;
```
Comment (tmccombs) [inline exercises/raindrops/raindrops.d]: this should be removed

**Comment by tmccombs** [comment]
I fixed some module names and imports after the files were renamed.

**Comment by kytrinyx** [comment]
Thanks! I've rebased onto master, so this should be ready to merge.

**Pull request merged**